Intros, Review, & Metadata (#3)
danielehc/katacoda-scenarios: consul_service_mesh_1/intro.md

# Consul Service Mesh on Kubernetes

In this lab you will perform all the steps needed to deploy Consul service mesh as the data plane for your Kubernetes cluster.

The lab uses Minikube to deploy the cluster but the commands can be applied to any Kubernetes flavor.

> If you are already familiar with the basics of Consul, [understand Consul service mesh](https://learn.hashicorp.com/consul/gs-consul-service-mesh/understand-consul-service-mesh) provides a reference guide for the Consul service mesh based labs.

Reviewer: ```suggestion
If you are already familiar with the basics of Consul, [Understand Consul Service Mesh](https://learn.hashicorp.com/consul/gs-consul-service-mesh/understand-consul-service-mesh) provides a reference guide for the Consul service mesh based labs.
```

> In this hands-on lab, you will deploy a Consul datacenter using Minikube and Helm 3.x. Specifically you will:

Reviewer 2: We just say `In this lab ` two statements above. Shall we remove one of the two? (no strong opinion anyway)

> In this hands-on lab, you will deploy a Consul datacenter using Minikube and Helm 3.x. Specifically you will:

Author: ```suggestion
During this session you will deploy a Consul datacenter using Minikube and Helm 3.x. Specifically you will:
```
